text/0800-ts-adoption-plan.md

```diff
@@ -316,7 +317,7 @@ Like browsers and Node, TypeScript regularly introduces new features which are a
 - `const` assertions and higher-order inference for functions, which was key to enabling Glint (3.4)
 - assertion functions, allowing more useful typing of `assert` (3.7)
 - the `declare` modifier, allowing safe declaration of injections and CPs (3.7)
-- spec compatibility for class fields, optional chaining,. and nullish coalescing (3.7)
+- spec compatibility for class fields, optional chaining,. andish coalescing (3.7)
```

Comment:
Is this change intended?

Reply:
I guess not: https://github.com/emberjs/rfcs-tooling/blob/4dd16f396c6b44d54face3114062f78143258b54/lib/frontmatter-updater.js#L16

Reply:
Yeah, looks like a bug!

Reply:
Was fixed in https://github.com/emberjs/rfcs-tooling/pull/15.